File: Java/01-Heads-Tails/src/HeadsTails.java
public class HeadsTails {

    public static void main(String[] args) {

        int result = (Math.random() <= 0.5) ? 1 : 2;

        if(result == 1)
            System.out.println("It's heads!");

        else
            System.out.print("It's tails!");
    }
}

File: JavaScript/01-Heads-Tails/HeadsTails.js
   var coin = ["Heads", "Tails"];

function coinResults(){

    var rand =Math.floor(Math.random() * coin.length);
    document.getElementById('resultDisplay').innerHTML = coin[rand];
}

// function tails(){

//     var rand2 =Math.floor(Math.random() * coin.length);
//     document.getElementById('resultDisplay').innerHTML = coin[rand2];  
// }